packer/install-dependencies.sh

```diff
@@ -126,6 +126,21 @@ for NVIDIA_DEVICE_PLUGIN_VERSION in ${NVIDIA_DEVICE_PLUGIN_VERSIONS}; do
     pullContainerImage "docker" "nvidia/k8s-device-plugin:${NVIDIA_DEVICE_PLUGIN_VERSION}"
 done
 
+TUNNELFRONT_VERSIONS="v1.9.2-v4.0.4"
```

Comment:
what does the dash mean in "v1.9.2-v4.0.4"? is it one version? or should it be a space?

Reply:
Talk to the publisher. :) It's just the way it's versioned.

Reply:
```
Containers:
  tunnel-front:
    Container ID:   docker://bbf2a60b072c724b88132537a0caf961c467c73790c8e44f12a9467ba402e85e
    Image:          docker.io/deis/hcp-tunnel-front:v1.9.2-v4.0.4
```

Reply:
Actually, given that this is already oddly versioned as you noticed, let's skip the whole "v" removal thing and just keep this as an oddball.

Reply:
sounds good